api/matlab/examples/Us4R_control_colorPwi.m

```diff
@@ -86,7 +86,10 @@
 
 %% Preview (continuous in-loop operation)
 display = DuplexDisplay(rec, 'dynamicRange',    [0 80], ...
-                             'powerThreshold',  20);
+                             'powerThreshold',  0, ...
```

Comment:
Add possible threshold ranges in comment here and in the docstring. 
Consider add some information describing these thresholds and advice on how to apply them.

Reply:
Some description added in this place only. 
Arrus documentations are a better place for tips how to use the thresholds.